test/sass/import-css.sass

```diff
@@ -1,7 +0,0 @@
-// Special behavior of node-sass/libsass with CSS-files
-// 1. CSS-files are not included, but linked with @import url(path/to/css)
-@import ~css/some-css-module.css
-// 2. It does not matter whether the CSS-file exists or not, the file is just linked
-@import ./does/not/exist.css
-// 3. When the .css extension is missing, the file is included just like scss
-@import ~css/some-css-module
```

Comment:
Why some tests will be removed?

Reply:
CSS imports are not currently supported by Dart Sass. CSS imports with the `.css` suffix are also slated for removal from LibSass.

Reply:
@nex3 can we left css tests only for `node-sass`, we remove this after upgrade `node-sass` to 5.

Reply:
I don't know how, since all these test files are shared between the two implementations.

Reply:
@nex3 need investigate, will be great if you do this, otherwise i can try this too

Reply:
I definitely don't know how to do it without a huge refactor of how tests are run.

Reply:
@evilebottnawi - do you have any ETA when you might be able to look at this? :) Or, is it possible to move forward without this, as the features being tested are soon to deprecate anyway?

Reply:
@TroyAlford need @jhnns review